src/bridge.js

```diff
@@ -3,13 +3,14 @@ import { EventEmitter } from 'events'
 export default class Bridge extends EventEmitter {
   constructor (wall) {
     super()
-    this.setMaxListeners(Infinity)
-    this.wall = wall
+    const self = this
```

Comment:
that's what was causing the error in the safari build: `ReferenceError: Cannot access uninitialized variable.`

Might be related to https://bugs.webkit.org/show_bug.cgi?id=171543